pkg/service/replay/replay.go

```diff
@@ -748,11 +753,12 @@ func (r *Replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 
 	// TODO Need to decide on whether to use global variable or not
 	verdict := TestReportVerdict{
-		total:   testReport.Total,
-		failed:  testReport.Failure,
-		passed:  testReport.Success,
-		ignored: testReport.Ignored,
-		status:  testSetStatus == models.TestSetStatusPassed,
+		total:    testReport.Total,
```

Comment:
Along with saving time taken per test-set, you can also save total time taken print it at last in the summary.